### ISSUE TITLE
exporting for specific section

### ISSUE THREAD

**Issue by Hachem-H** [musescore/MuseScore#21516]
### Your idea

In musescore we can separate scores by sections, say in a classical setting, perhaps a multi-movement string quartet can be divided by sections. It would be quite useful to be able to export audio (possibly pdf files too) of that specific section alone.

### Problem to be solved

In those cases I tend to export the entire score (which at times can be upwards of an hour long) and I have to move the entire audio file into audacity then cut it up to finally export the section I want.
This idea can fix the struggle of doing all that.

**Comment by wizofaus** [comment]
Yes please!  If you have a range selected, having the ability to export just that range as audio (or anything really, though PDF might be a bit weird, given the layout would necessarily change) would be super useful.
